### PR TITLE
Balaca86 is adding a file for Chennai, TN

### LINKED ISSUE #14: Add a file for balaca86 Chennai
# Adding file for Chennai
- [x] Create a branch
- [x] Add a file
- [x] Commit my changes
- [x] Open a pull request
- [x] Code review
- [ ] Merge the pull request

### PR DESCRIPTION
**Pull request by balaca86**
This pull request resolves #14 

@mahas4u where else would you like to eat and visit?

### CODE REVIEW

File: balaca86-chennai.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Chennai, TN
+
+
+## Great places to eat
+- Hotel Saravana Bhavan
+- Residency Towers
+- Bikes and Barrels
+## Things to do
+- Marina Beach
```
Comment (leefp) [inline balaca86-chennai.md]: I wish I was at a beach, right about now....

File: balaca86-chennai.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- Catamaran ride @ Fisherman's Cove
+- Mylapore Kabaleeshwara Temple
```
Comment (leefp) [inline balaca86-chennai.md]: I haven't been to this temple before....

File: balaca86-chennai.md (hunk 1 above)
Comment (balaca86) [inline balaca86-chennai.md]: @leefp It is the natural urban beach which is 4 mi long, which makes it India's longest beach!!

Comment (leefp) [inline balaca86-chennai.md]: very cool....although may be a bit far for me, at the moment.

**Pull request merged**